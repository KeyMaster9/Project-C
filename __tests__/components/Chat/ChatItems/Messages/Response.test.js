import React from 'react';
import renderer from 'react-test-renderer';

beforeEach(() => {
    jest.restoreAllMocks();
});

it('should render a TravelAgent Component', () => {
    const { default: Response } = require('../../../../../src/components/Chat/ChatItems/Messages/Response.jsx');
    const component = renderer.create(<Response message='mock response message'/>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});