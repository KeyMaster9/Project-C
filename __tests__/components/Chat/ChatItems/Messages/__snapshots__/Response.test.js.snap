// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a TravelAgent Component 1`] = `
<div
  className="MessageContainer"
>
  <div
    className="Message Response"
  >
    <p>
      mock response message
    </p>
  </div>
</div>
`;
